pkg/log/logger.go

```diff
@@ -121,3 +122,12 @@ func String(key, val string) zap.Field {
 	}
 	return zap.String(key, val)
 }
+
+type DebugLogger struct {
```

Comment:
It looks more like `PrefixedLogger`, doesn't it?

Reply:
Renamed: https://github.com/aquasecurity/trivy/pull/5550/commits/d9389f779f184a58e7d9ecdf33b8f043751642f2